core/src/main/resources/lib/layout/card.jelly

```diff
@@ -0,0 +1,58 @@
+<!--
+The MIT License
+
+Copyright (c) 2024 Jan Faracik
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout">
+  <st:documentation>
+    <st:attribute name="title" use="required">
+        Title of the card
+    </st:attribute>
+    <st:attribute name="id">
+        Optional ID for the card
+    </st:attribute>
+    <st:attribute name="controls">
+      Shows controls in the top right of the card
+    </st:attribute>
+    <st:attribute name="expandable">
+      Shows the expand icon if set, navigates to the value provided on click
+    </st:attribute>
+  </st:documentation>
+
+  <div class="jenkins-card" id="${attrs.id}">
+    <div class="jenkins-card__title">
+      ${attrs.title}
+      <div class="jenkins-card__controls">
+        <j:out value="${controls}"/>
+        <j:if test="${attrs.expandable != null}">
+          <a href="${attrs.expandable}" class="jenkins-card__reveal" tooltip="${%Expand}">
```

Comment:
I wonder if this tooltip should be able to be overridden, in this case I'm not sure expand is the best word.

I think something like 'Build time trend' maybe? its not really expanding in this case

Reply:
I'd like to update the 'Build time trend' page eventually to behave more like BlueOceans builds page - but that could take a while still.

Happy to change the tooltip if you feel I should.

Reply:
limitation is that core is not really able to work well with pipeline jobs. So I wrote this plugin https://plugins.jenkins.io/pipeline-agent-build-history/

Reply:
It’s not a blocker just a thought